media/js/src/utils.js

```diff
@@ -358,6 +358,9 @@ const displayGraphType = function(gType) {
         case 21:
             name = 'Optimal Choice: Cost-Minimizing Production Inputs';
             break;
+        case 22:
+            name = 'Taxation and Tax Revenues';
```

Comment:
This graph name should be the same as what's in the django back-end, which is "Tax Rate and Revenue"